app/src/main/res/layout/all_topics.xml

```diff
@@ -16,15 +16,14 @@
 
```

Comment:
Also,

there is `<data> ... </data>` at top of this file which is not getting used anywhere.
I think it was missed in Low-fi PR, so can your remove that too from here?

Reply:
removed.